UI/CombatReport/CombatLogWnd.cpp

```diff
@@ -187,6 +280,79 @@ namespace {
         SetCollapsed(new_collapsed);
     }
 
+    /// A section used in initial forces and destroyed forces, displays empire's name and forces count;
+    /// and can be expanded to enumerate all forces of this empire that take part in the combat.
+    class EmpireForcesAccordionPanel : public AccordionPanel {
+    public:
+        EmpireForcesAccordionPanel(GG::X w,
+                                   CombatLogWnd::Impl& log_,
+                                   int viewing_empire_id_,
+                                   int empire_id,
+                                   EmpireForces forces_);
+
+        ~EmpireForcesAccordionPanel();
```

Comment:
should probably be virtual (even though equivalent code doesn't do so...)

Reply:
I'm fairly sure, that destructors of classes derived from types with virtual destructors have automatically virtual destructors too, but I'll check with the standard later.

Reply:
They are, but marking the derived classes is sometimes done for documentation reasons, I gather...

Reply:
Either way, I was mimicking local code style (for lack of better references). If you believe, that `virtual` should be always explicit for virtual destructors, I will make it so.

Reply:
It's not very important... Since there are other cases of similar code that don't, then being consistent with the surrounding style is fine.